### DISCUSSION TITLE
Trojan server detected

### DISCUSSION THREAD

**Discussion by uugan** [rejetto/hfs#655] [General]
Trojan & backdoor service detected, attacker can access to shell and run scripts for getting LDAP users
![image](https://github.com/rejetto/hfs/assets/601881/634af50f-d6e1-4ec4-b7ab-0a0ca331788e)

**Comment by rejetto** [comment]
hey, 
you didn't specify what version you scanned, and what software/website you used to scan.
Anyway, it probably doesn't matter, because i believe it's a false positive, and the reason is written in this page:
https://github.com/rejetto/hfs/wiki/Antivirus

the fact alone that at the top of the page says "node.exe" is a big clue that the warnings are wrong.

**Comment by uugan** [comment]
This was latest version of windows. Downloaded and checked from virustotal. 

Below version was installed on our server(i'm not sure which version it was, but year downloaded ~2017) and our security guys detected attack.
https://www.virustotal.com/gui/file/93019ff4c7f345b6b03ada2c60efc51f0f199f5356d8bf1b85cd9649420fa84f/detection
![image](https://github.com/rejetto/hfs/assets/601881/10f52346-4793-4b65-b443-915157ff4faa)

**Comment by rejetto** [comment]
i see in you screenshot 559KB.
HFS is 40-50MB, of course the file you are using is not the correct one you download from here.
where did you get the file?

and saying "latest version" is not ok here, say the exact number.
